index.rst

```diff
@@ -1,51 +1,517 @@
+As shown in :numref:`fig-sim_image` below, these images contain a moderately crowded field of stars (no galaxies) with Kolmogorov double-gaussian PSFs, realistic SEDs, photon shot noise, and no other effects other than DCR.
+In this example, the model is built using three frequency planes and eight input simulations of the field, with airmass ranging between 1.0 and 2.0 (*not* including the simulated observation shown in :numref:`fig-sim_image`).
+I use :eq:`eqn-basic_template` to build a DCR-matched template for the simulated observation (:numref:`fig-sim_template`), and subtract this template to make the difference image (:numref:`fig-sim_template_diff`).
+For comparison, in :numref:`fig-sim_image_diff` I subtract a second simulated image generated with the same field 10 degrees closer to zenith.
+Note that the noise in :numref:`fig-sim_template_diff` is :math:`\sim \sqrt{2}` lower than in :numref:`fig-sim_image_diff`, because the template is a form of coadded image with significantly lower noise than an individual exposure.
+
+.. figure:: /_static/simulations/simulated_108_image.png
+   :name: fig-sim_image
+
+   Simulated g-band image with airmass 1.3
+
+.. figure:: /_static/simulations/simulated_image_108_template.png
+   :name: fig-sim_template
+
+   The DCR-matched template for the simulated image in :numref:`fig-sim_image`.
+
+.. figure:: /_static/simulations/simulated_image_108_template_difference.png
+   :name: fig-sim_template_diff
+
+   Image difference of the simulated image :numref:`fig-sim_image` with its template from :numref:`fig-sim_template`.
+
+.. figure:: /_static/simulations/simulated_image_108_112_difference.png
+   :name: fig-sim_image_diff
+
+   Image difference of :numref:`fig-sim_image` with another simulation of the same field 10 degrees closer to zenith (airmass 1.22).
+
+Dipole mitigation
+^^^^^^^^^^^^^^^^^^
+
+A quantitative assessment of the quality of the template from :numref:`fig-sim_template` is the number of false detection in the image difference, :numref:`fig-sim_template_diff`.
+Since there are no moving or variable sources in these simulated images, any source detected in the image difference is a false detection.
+From comparing  :numref:`fig-sim_template_diff` and :numref:`fig-sim_image_diff` it is clear by eye that the DCR-matched template produces fewer dipoles, and we can see in :numref:`fig-dcr_dipoles` that this advantage holds regardless of airmass or elevation angle.
+The DCR-matched template appears to perform about as well as an exposure taken within 5 degrees of the science image, but recall that the noise - and therefore the detection limit - is higher when using a single image as a template.
+
+
+.. figure:: /_static/simulations/DCR_dipole_plot.png
+   :name: fig-dcr_dipoles
+
+   Plot of the number of false detections for simulated images as a function of airmass, for different templates.
```

Comment:
Can you comment on the number of false detections expected from pure noise, given the size of your images?

Reply:
I believe there should be ~1 false detection due to noise. I've added a comment.